Limelight/Stream/VideoDecoderRenderer.m

```diff
@@ -106,38 +118,37 @@ - (void)start
         _displayLink.preferredFramesPerSecond = self->frameRate;
     }
     [_displayLink addToRunLoop:[NSRunLoop mainRunLoop] forMode:NSDefaultRunLoopMode];
+
+}
+
+- (void) setupDecompressionSession {
+    if (decompressionSession != NULL){
+        VTDecompressionSessionInvalidate(decompressionSession);
```

Comment:
it is also necessary to do the following:
VTDecompressionSessionWaitForAsynchronousFrames(decompressionSession);